Client/src/components/contanier/student/Coures/index.js

```diff
@@ -1,28 +1,46 @@
 import React, { Component } from 'react'
+import { Redirect, Link } from 'react-router-dom'
+import axios from 'axios'
 
 import Course from '../../../common/Course'
 import Button from '../../../common/Button'
-import data from './staticData'
 import './index.css'
 
 export default class Courses extends Component {
 
-  handleBtnClick = (e) => {
-    e.preventDefault()
+  state = {
+    data: null,
+  }
+
+  componentDidMount() {
+    if (this.props.level) {
+      // ${this.props.level + 1}
```

Comment:
here we already stored the target level so u shouldn't increment the level number with 1